batch/batch/front_end/front_end.py

```diff
@@ -1228,10 +1255,10 @@ async def insert(tx):
                 billing_project,
                 json.dumps(attributes),
                 batch_spec.get('callback'),
-                batch_spec['n_jobs'],
+                0,
                 now,
                 token,
-                'open',
+                'complete',
                 BATCH_FORMAT_VERSION,
                 batch_spec.get('cancel_after_n_failures'),
```

Comment:
I think this is something that I accidentally left out from your original changes: the `time_completed` should be set here to `now`